connector/dummy_simulate.go

```diff
@@ -217,8 +217,7 @@ func (conn *Dummy) MessageDeleted(messageID imap.MessageID) error {
 }
 
 func (conn *Dummy) UIDValidityBumped() {
```

Comment:
Do we still need this?

Reply:
Yes. it is still used to test UID validity bumped or if we need to force GLuon to regenerate for whatever reason.